mlir/utils/jenkins/Dockerfile

```diff
@@ -135,5 +139,5 @@ RUN python3 -m venv /tuna-venv && . /tuna-venv/bin/activate && \
     python3 -m pip install -r tuna-requirements.txt --ignore-installed && \
     python3 -m pip install scipy pandas
 
-# Workaround for 6.0.0 rocprof not supporting navi3x:  add it to list.
+# Workaround for 6.1.1 rocprof not supporting navi3x:  add it to list.
```

Comment:
Issue [#1496](https://github.com/ROCm/rocMLIR-internal/issues/1496) addresses this hack.  I have a prototype solution that needs a little work, and a way to measure bank conflicts.